R/temporalBehaviour.R

```diff
@@ -425,7 +431,12 @@ temporalBehaviour <- function(sts,
                               empiricalRMW = FALSE,
                               tempRes = 1,
                               verbose = 1) {
-  checkInputsTemporalBehaviour(sts, points, product, windThreshold, method, asymmetry, empiricalRMW, tempRes, verbose)
+  
+  if(is.null(windThreshold)){
+    windThreshold = sshs
```

Comment:
Not coherent with defaut value in function call....
Maybe test that case that if NULL, then we should throw an error if `EXPOSURE` is requested

Reply:
Done